Documentation/ART_NOTES.md

```diff
@@ -0,0 +1,119 @@
+
+### **Lighting set up for the MainMenu, CharSelect, and PostGame Scenes:**
+
+* To bake these scenes, open them and navigate to the CharSel_Background Prefab in the scene. In the prefab, there is are two other prefabs called "EnvLighting_Realtime" and "EnvLighting_ForBaking_" with lights set up for realtime, and lights set up for baking. Leave the Realtime prefab enabled if you're using realtime lighting, otherwise, disable it and enable the Baking prefab when baking lighting.
+
+    ![CharSelectLights](Images/CharSelectLights.png) <br /><br />
+
+### **Lighting set up for the BossRoom Scene:**
+
+* The main Boss Room level is separated into three scenes (Assets > BossRoom > Scenes > BossRoom):
+  * Dungeon BossRoom
+  * Dungeon Entrance
+  * Dungeon Transition <br /><br />
+
+* There are two prefabs for the environment lights (Assets > Prefabs > Dungeon):
+  * EnvLighting_ForBaking
+  * EnvLighting_ForRealtime <br /><br />
+
+* Drag whichever one you need into your scene, and make sure it's at (0,0,0) for position and rotation, (1,1,1) for scale to ensure the lights are in the right position. <br /><br />
+
+* The env_torch2 prefab, used throughout the Boss Room scene, also has both a realtime light and a baking light that you can choose from.
+
+    ![TorchPrefab](Images/TorchPrefab.png) <br /><br />
+
+To bake, open the BossRoom scene. Load the pieces of the dungeon listed above into that scene, make sure you're using the baking version of all of the lights, then generate your lighting. <br /><br />
+
+
+## **Resources from Unity for more details:**
+  * [How to build Lightmaps in Unity 2020.1 | Tutorial](https://youtu.be/KJ4fl-KBDR8) (this is a short overview video on baking)
+  * [Harnessing Light with URP and the GPU Lightmapper | Unite Now 2020](https://youtu.be/hMnetI4-dNY) (this video is quite long, but runs through everything you'd want to know about baking, light probes, UVs, etc. Check out the comments for more tips and tricks, too)
+  * [Unity Manual: Lightmappers](https://docs.unity3d.com/Manual/Lightmappers.html)
+  * [Unity Manual: Light Modes](https://docs.unity3d.com/Manual/LightModes.html)
+  * [Unity Manual: Light Probes](https://docs.unity3d.com/Manual/LightProbes.html)
+  * [Unity Manual: Lighting Window](https://docs.unity3d.com/Manual/lighting-window.html)
+  * [Unity Manual: Lighting Settings](https://docs.unity3d.com/Manual/class-LightingSettings.html)
+  * [Unity Manual: Lighting Explorer](https://docs.unity3d.com/Manual/LightingExplorer.html)
+  * [Unity Manual: Lightmapping Directional](https://docs.unity3d.com/Manual/LightmappingDirectional.html)
+  * [Unity Manual: Reflection Probes](https://docs.unity3d.com/Manual/class-ReflectionProbe.html)
```

Comment:
Fantastic. I feel like we went above and beyond for the art documentation, especially with these resources. Good job!